modules/creating-a-machine-pool-ocm.adoc

```diff
@@ -86,6 +86,10 @@ endif::openshift-rosa[]
 .. Expand the *Edit node labels and taints* menu.
 .. Under *Node labels*, add *Key* and *Value* entries for your node labels.
 .. Under *Taints*, add *Key* and *Value* entries for your taints.
+[NOTE]
+====
+Creating a machine pool with taints is only possible if the cluster already has at least one machine pool without a taint.
```

Comment:
Hmm... In theory a cluster will always to have a machinepool without taint. So customer can create machinepool with taint anytime.